examples/activity-analyzer/src/tests/index.ts

```diff
@@ -1,32 +1,39 @@
-import "mocha";
-import {core, helpers} from "@mediarithmics/plugins-nodejs-sdk";
-import {MyActivityAnalyzerPlugin} from "../MyPluginImpl";
+import 'mocha';
+import {MyActivityAnalyzerPlugin} from '../MyPluginImpl';
+import {ActivityAnalyzer, PluginProperty} from '@mediarithmics/plugins-nodejs-sdk/lib/mediarithmics';
```

Comment:
You changed the import path of the lib -> everything was exported at the root of the lib.

Can you add it in the MIGRATION.md / breaking changes listing?

Also:
- are you sure  about having "lib" in all the import paths?
- I get the "/helper" for the test helpers. But a "/mediarithmics" for eveyrthing else? That's a lot of mediarithmics: @**mediarithmics**/plugins-nodejs-sdk/lib/**mediarithmics** don't you think?